test/ibmq/test_experiment.py

```diff
@@ -604,15 +645,17 @@ def test_analysis_results_limit(self):
     def _create_experiment(
             self,
             backend_name: Optional[str] = None,
-            start_dt: Optional[datetime] = None
+            start_dt: Optional[datetime] = None,
+            share_level: Optional[Union[ExperimentShareLevel, str]] = None
     ) -> Experiment:
         backend_name = backend_name or self.experiments[0].backend_name
         new_exp = Experiment(
             provider=self.provider,
             backend_name=backend_name,
             experiment_type='test',
             tags=['qiskit-test'],
-            start_datetime=start_dt
+            start_datetime=start_dt,
+            share_level=share_level
```

Comment:
I can't figure out why this is failing:

https://github.com/Qiskit/qiskit-ibmq-provider/pull/858/checks?check_run_id=1664935530

test/ibmq/test_experiment.py:652:18: E1123: Unexpected keyword argument 'share_level' in constructor call (unexpected-keyword-arg)

It doesn't fail for me locally.